core/services/fluxmonitor/flux_monitor.go

```diff
@@ -661,15 +661,28 @@ func (p *PollingDeviationChecker) SetOracleAddress() error {
 			}
 		}
 	}
+
+	loggerFields := []interface{}{
```

Comment:
I think a cleaner way is with logger.With actually see [https://github.com/smartcontractkit/chainlink/blob/8640440bcb897db63171fc243531ca6dcd026648/core/services/fluxmonitorv2/flux_monitor.go#L699]

Reply:
Updated